Source/SwiftLintFramework/Rules/Metrics/CyclomaticComplexityRule.swift

```diff
@@ -12,22 +12,75 @@ public struct CyclomaticComplexityRule: ASTRule, ConfigurationProviderRule {
         description: "Complexity of function bodies should be limited.",
         kind: .metrics,
         nonTriggeringExamples: [
-            "func f1() {\nif true {\nfor _ in 1..5 { } }\nif false { }\n}",
-            "func f(code: Int) -> Int {" +
-                "switch code {\n case 0: fallthrough\ncase 0: return 1\ncase 0: return 1\n" +
-                "case 0: return 1\ncase 0: return 1\ncase 0: return 1\ncase 0: return 1\n" +
-                "case 0: return 1\ncase 0: return 1\ndefault: return 1}}",
-            "func f1() {" +
-            "if true {}; if true {}; if true {}; if true {}; if true {}; if true {}\n" +
-                "func f2() {\n" +
-                    "if true {}; if true {}; if true {}; if true {}; if true {}\n" +
-                "}}"
+            """
+            func nonTriggering() {
+                if true {
+                    for _ in 1..5 {
+                    }
+                }
+                if false {
+                }
+            }
+            """,
+            """
+            func switchFunction(code: Int) -> Int {
+                switch code {
+                case 0: fallthrough
```

Comment:
can you please change the `case`s to avoid repetition as we're changing them anyway?